apiserver/facades/client/applicationoffers/base.go

```diff
@@ -431,75 +432,85 @@ func (api *BaseAPI) makeOfferParams(backend Backend, offer *jujucrossmodel.Appli
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
-	appBindings, err := app.EndpointBindings()
-	if err != nil {
-		return nil, nil, errors.Trace(err)
-	}
+
```

Comment:
Changes to this file are more than name to id changes, they include rework of the method with those changes.  Please take a look